build/FUSE_SRC/FUSE_NETCDF/get_gforce.f90

```diff
@@ -12,6 +12,33 @@ module get_gforce_module
 
 contains
 
+ subroutine split_dims(nSpat2, proc, nproc, start)
+     implicit none
+     integer(i4b), intent(inout) :: nSpat2, start
+     integer(i4b), intent(in) :: proc, nproc
+
+     integer(i4b) :: newn, offset, even_multiple, count
+
+     newn = nSpat2 / nproc
+
+     print*, newn, nproc
```

Comment:
Might want to drop these print statements or make them into more meaningful statements (akin to the other print statements in fuse.